tests/sbr-operator/tests/must_gather.go

```
@@ -96,13 +96,26 @@ var _ = Describe(
 					GinkgoWriter.Printf("Warning: failed to write collected-paths.txt: %v\n", writeErr)
 				}
 
-				By("Validating node YAMLs for all cluster nodes")
+				By("Validating node YAMLs for cluster nodes")
 
+				var missingNodes []string
 				for _, nodeName := range nodeNames {
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (no shared variables above range)

(wsl_v5)